REVIEWING.md

```diff
@@ -0,0 +1,66 @@
+
+This document is meant to be a set of tips and guidelines for reviewers of pull requests to the project.
+For general information about the project, see [our readme](https://github.com/zeldaret/mm/blob/master/README.md).
+
+Most discussions happen on our [Discord Server](https://discord.zelda64.dev) where you are welcome to ask if you need help getting started, or if you have any questions regarding this project and other decompilation projects.
+
+Useful Links
+------------
+
+- [Discord](https://discord.zelda64.dev/) - Primary discussion platform.
+- [Trello board](https://trello.com/b/ruxw9n6m/majoras-mask-decompilation) - We use this to track decompilation progress, not GitHub Issues.
+
+- [Contributing guide](https://github.com/zeldaret/mm/blob/master/CONTRIBUTING.md) - Instructions for contributors.
+- [Installation guide](https://github.com/zeldaret/mm/blob/master/README.md#installation) - Instructions for getting this repository set up and built on your machine.
+
+- [Zelda 64 Reverse Engineering Website](https://zelda64.dev/) - Our homepage, with links to other resources.
+- [Introduction to OOT decomp](https://github.com/zeldaret/oot/blob/master/docs/tutorial/contents.md) - A very detailed tutorial on how to get started with decomp. For OOT, but largely applicable to MM as well.
+- The `#resources` channel on the Discord contains many more links
+
+Getting Started
+---------------
+
+### What should I know to take part in the review process?
+
+You should first get yourself familiar with our [Contributing guide](https://github.com/zeldaret/mm/blob/master/CONTRIBUTING.md). It is also recommended that you have already successfully submitted a merged pull request to understand how the process works before submitting a review.
+
+Pull Requests (PRs)
+------------
+
+### How to review a Pull Request
+
+- When reviewing a PR it is suggested that you follow the checklist outlined in this section.
+- You are not required to check for all of the items in the checklist. It is meant as a guide.
+- Once the PR author has addressed all of your comments, you should add a comment with approval to the PR to signify to the project leads that this PR has been through peer review.
+- If someone does not address your comments and expresses that a different way is better than yours, look for feedback from other contributors. The project lead will have final say in these situations. All decisions will generally be guided by a consensus of contributors.
+
+### Reviewer Checklist
+- [ ] Jenkins build is successful.
+   - [ ] `make` builds a matching ROM.
+   - [ ] Any new compiler warnings that were added are required for matching. Ensure there is good reason if the warnings files have changed. `./tools/warnings_count/warnings_build_current.txt` and `./tools/warnings_count/warnings_setup_current.txt`
+- [ ] Files with NON_MATCHING functions have equivalent behaviour.
```

Comment:
I left a comment but I think it got lost - is there something to look for with `NON_EQUIVALENT` sections?

Reply:
Ideally we don't want anything `NON_EQUIVALENT` to get in unless several people have looked at it and can't fix it. so I'd have thought something like
```
- [ ] Any NON_EQUIVALENT functions have been discussed in Discord.
```
would be the best (while it's true that a PR is generally the place to discuss things, this avoids people marking stuff as `NON_EQUIVALENT` just because they're fed up with the file and want to make it look like they've looked at everything). `NON_EQUIVALENT` is to be used as a last resort, essentially.

Reply:
I like that wording a lot.

Reply:
What about dropping the `in Discord` part?
```md
- [ ] Any NON_EQUIVALENT functions have been previously discussed.
```

Reply:
I liked the "in Discord" bit because it implied it should be done *before* opening the PR. Maybe combining it with yours is even stronger?

```
- [ ] Any NON_EQUIVALENT functions have been previously discussed in Discord.
```

Reply:
I would say that generally NON_EQUIVALENT functions should just be left as asm. To me, it's the same thing as wrapping it around an `#if 0`

Reply:
I'll also add that you should at least be attempting every function in the file.